static-assets/components/cstudio-dialogs/select-content-type.js

```diff
@@ -304,13 +296,13 @@ CStudioAuthoring.Dialogs.DialogSelectContentType = {
     $('#wcm-content-types-dropdown').fadeIn('fast');
   },
 
-  getImage(path) {
+  getImage(contentTypeId) {
     const qs = CrafterCMSNext.util.object.toQueryString({
-      site: CStudioAuthoringContext.site,
-      path
+      siteId: CStudioAuthoringContext.site,
+      contentTypeId
     });
 
-    return CrafterCMSNext.util.ajax.getBinary(`/studio/api/1/services/api/1/content/get-content-at-path.bin${qs}`);
+    return CrafterCMSNext.util.ajax.getBinary(`/studio/api/2/configuration/content-type/preview_image${qs}`);
```

Comment:
Make this call into a service (services/contentTypes.ts) and use that all across.